layouts/partials/tabs.html

```diff
@@ -60,9 +60,12 @@ <h3 class="subtitle is-5 is-muted">{{ $subtitle }}</h3>
 
         <ul class="cd-tabs__panels">
         <li id="tab-inbox" class="cd-tabs__panel cd-tabs__panel--selected text-component">
-            <p>Visualization Lorem ipsum dolor sit amet, consectetur adipisicing elit. Earum recusandae rem animi accusamus quisquam reprehenderit sed voluptates, numquam, quibusdam velit dolores repellendus tempora corrupti accusantium obcaecati voluptate totam eveniet laboriosam?</p>
-
-            <p>Inbox Lorem ipsum dolor sit amet, consectetur adipisicing elit. Earum recusandae rem animi accusamus quisquam reprehenderit sed voluptates, numquam, quibusdam velit dolores repellendus tempora corrupti accusantium obcaecati voluptate totam eveniet laboriosam?</p>
+                <img src="images/content_images/visualization.gif" alt="visualization figure" align="left" height="300" width="300" style="margin: 10px 25px; border-radius: 6px"> 
+                <p>Data Visualization refers to graphical representation of textual data that makes it human-readable, easier to understand and interpret by exposing patterns, trends and correlations within the data. Data visualization paves the way towards absorbing data and gaining actionable insights. It enables advanced data analytics for identifying areas that need attention and improvement. Users can make data-driven decisions and analyse huge volumes of data through visual elements such as graphs, bar, pie and line charts, maps, infographics, dashboards, geographic maps, heatmaps etc.</p>
+                <p>Data Scientists rely upon data visualization to manipulate, analyse and query large volumes of data using interactive images, visual analytics and gain useful insights into the data that cannot be otherwise detected in textual form.</p>
+                <p><mark>NumPy</mark> is the key building block of all heavyweight Python data processing and visualization libraries. Python’s native performance with large quantities of data is relatively slow, but NumPy, implemented by low-level libraries written in C and Fortran, performs parallel operations on large arrays all at once, making data-processing and visualization very fast.</p>
+<hr>
+<p>Image Source: https://www.pinterest.com/pin/634866878694365512/</p>
```

Comment:
perhaps we can steal a prettier image from a gallery of one of the packages I just mentioned above?

Reply:
Done, used one from Bokeh